examples/rossmann/rossmann-store-sales-fastai.ipynb

```diff
@@ -296,6 +297,7 @@
     "from fastai.basics import Learner\n",
     "from fastai.basics import MSELossFlat\n",
     "from fastai.callback.progress import ProgressCallback\n",
+    "from fastai.metrics import APScoreMulti, RocAucMulti\n",
```

Comment:
Are these imports used anywhere? I don't see it reference in the rest of this PR